client/src/components/container/portal/LogInPage/index.js

```diff
@@ -1,63 +1,71 @@
 import React, { Component } from 'react';
 import { Form, Input, Button, Spin, Alert, message } from 'antd';
 import axios from 'axios';
-
+import propTypes from 'prop-types';
 import './style.css';
 
-export class PortalLogin extends Component {
+export default class PortalLogin extends Component {
   state = {
     isLoade: false,
-    pinCode: '',
-    serverError: '',
+    isFinish: false,
     msg: '',
     error: false,
   };
 
-  handleChange = ({ target: { value } }) => {
-    this.setState({ pinCode: value, error: false });
+  formRef = React.createRef();
+
+  onFinish = async ({ pinCode }) => {
+    const {
+      props: {
+        history: { push },
+      },
+    } = this;
+    this.setState({ isLoade: true, isFinish: true });
+    try {
+      const { data } = await axios.post('/api/v1/portal/login', { pinCode });
+      push('/portal/attendance');
+      message.success(data.msg, 10);
+    } catch ({
+      response: {
+        data: { msg },
+      },
+    }) {
```

Comment:
reset input field should be inside catch : 
```suggestion
  onFinish = async ({ pinCode }, resetFields) => {
    const {
      props: {
        history: { push },
      },
    } = this;
    this.setState({ isLoade: true, isFinish: true });
    try {
      const { data } = await axios.post('/api/v1/portal/login', { pinCode });
      push('/portal/attendance');
      message.success(data.msg, 10);
    }  catch (err) { 
     let error; 
     resetFields();
     if (err.response) { 
       error = err.response.data.msg; 
     } else { 
       error = 'Something went wrong, please try again later'; 
     } 
     this.setState({ error, loading: false }); 
   } 
```